UltimatePDF_ExternalLogic/Management/Troubleshooting/Logger.cs

```diff
@@ -1,21 +1,26 @@
-﻿using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.Logging.Abstractions;
-using System;
+﻿using System;
 using System.Collections.Generic;
-using System.Diagnostics;
 using System.IO;
 using System.IO.Compression;
 using System.Text;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using UltimatePDF_ExternalLogic.Utils;
 
 namespace OutSystems.UltimatePDF_ExternalLogic.Management.Troubleshooting {
     public class Logger {
-        private static readonly string version = "UltimatePDF_ExternalLogic v0.1.6";
```

Comment:
Not something that needs addressing now but just wondering if we have a more robust way of setting the version string (as hardcoding may lead to this not being updated in the future)

Reply:
I have the version now in the file resources/version because of that